src/sir-robin.py

```diff
@@ -0,0 +1,58 @@
+import os
+from pathlib import Path
+
+import disnake
+from dotenv import load_dotenv
+from loguru import logger
+from disnake.ext import commands
+
+from util.constants import walk_extensions
+
+path = Path(__file__)
+parent = path.parents[1]
+load_dotenv(parent.joinpath(".env"))
+DEV_LOG = os.environ.get("DEV_LOG")
+TOKEN = os.environ.get("TOKEN")
+
+
+class SirRobin(commands.Bot):
+    """
+    Bradbot core.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.name = "Bradbot"
+        self.dev_log = int(DEV_LOG)
+
+    def add_cog(self, cog: commands.Cog) -> None:
+        """
+        Delegate to super to register `cog`.
```

Comment:
First line should also be on the same line as the opening triple quote iirc

Reply:
The style in the other projects is like this through flake8-docstrings, with an empty line after it.